include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -59,7 +59,7 @@ typedef void *indexstoredb_symbol_location_t;
 typedef void *indexstoredb_symbol_relation_t;
 typedef void *indexstoredb_unit_info_t;
 
-typedef enum {
+typedef enum : uint64_t {
```

Comment:
This is only valid in C23 or later (otherwise it's a C++ extension). Clang and gcc will allow that, but I don't think MSVC does unless you're in C++ mode.

I'm not sure if there is actually a portable way to get a sized enum before C23. May need to switch to a typedef of `uint64_t`.

Reply:
I don't think we actually use this from C anyway, only from Swift. It's theoretically possible that there's a client that is using it directly, but let's try this for now and then convert to typedef + enum + macro for the > 32 bit case if we end up having to.